news/xontrib-homebrew.rst

```diff
@@ -0,0 +1,23 @@
+**Added:**
+
+* <news item>
+
+**Changed:**
+
+* Replaced `xontrib-linuxbrew <https://github.com/eugenesvk/xontrib-linuxbrew>`_ with `xontrib-homebrew <https://github.com/eugenesvk/xontrib-homebrew>`_, which also supports Homebrew on macOS
+
```

Comment:
in .rst double ` is used for highlight.

Reply:
There is `_` at the end to make it a url

\`Link text \<link URL\>\`_

Reply:
Yes True, then  test_news.py  needs update.

![image](https://user-images.githubusercontent.com/6702219/126893476-669c5e14-de9c-4e89-9858-6b94645ce7c8.png)

Reply:
Hey @eugenesvk -- for the moment, can you remove the first link?  The RST checker in our CI isn't super advanced and this is tripping it up.  We should fix that, but it shouldn't block this.

Reply:
...and here I've almost added a temp fix of removing proper links first (so that the next `single_grave_reg` wouldn't trip over them)
```python
rst_quote_reg = re.compile(r"[^`]`[^`]+`[_]") # RST quote
l = re.sub(rst_quote_reg, '', l) # remove legitimate RST `quotes`_ to avoid errors later
```
But obviously @jnoortheen's solution of using a proper linter is the proper way to go!
Thanks for the fix!